src/tracks/InstrumentTrack.cpp

```diff
@@ -1098,8 +1098,10 @@ InstrumentTrackWindow * InstrumentTrackView::topLevelInstrumentTrackWindow()
 void InstrumentTrackView::createFxLine()
 {
 	int channelIndex = gui->fxMixerView()->addNewChannel();
+	auto channel = Engine::fxMixer()->effectChannel(channelIndex);
 
-	Engine::fxMixer()->effectChannel( channelIndex )->m_name = getTrack()->name();
+	channel->m_name = getTrack()->name();
+	if (getTrack()->useColor()) channel->setColor (getTrack()->color());
```

Comment:
Minor code convention issue (even one liners require explicit blocks):
```suggestion
	if (getTrack()->useColor()) { channel->setColor (getTrack()->color()); }
```

That's what was referred earlier as "Same as above", for some reason only the last comment was submitted.

Reply:
Didn't know this, thanks for pointing it out.